src/hotspot/share/opto/node.cpp

```diff
@@ -664,55 +664,51 @@ void Node::destruct(PhaseValues* phase) {
   }
 }
 
-//------------------------------grow-------------------------------------------
-// Grow the input array, making space for more edges
-void Node::grow(uint len) {
+//------------------------------resize_array-------------------------------------------
+ // Resize input or output array to grow it the next larger power-of-2 bigger
+// than len.
+void Node::resize_array( Node**& array, node_idx_t& max_size, uint len, bool is_in) {
   Arena* arena = Compile::current()->node_arena();
-  uint new_max = _max;
+  uint new_max = max_size;
   if( new_max == 0 ) {
```

Comment:
```suggestion
  if(new_max == 0) {
```
While we are here we might as well fix this too.